plugins/plugin-git/che-plugin-git-ext-git/src/main/java/org/eclipse/che/ide/ext/git/client/plugins/EditorTabsColorizer.java

```diff
@@ -80,7 +80,12 @@ public void onGitStatusChanged(String endpointId, StatusChangedEventDto statusCh
         .get()
         .getOpenedEditors()
         .stream()
-        .filter(editor -> editor instanceof HasVcsChangeMarkerRender)
+        .filter(
+            editor ->
+                editor instanceof HasVcsChangeMarkerRender
+                    && statusChangedEventDto
+                        .getProjectName()
+                        .equals(editor.getEditorInput().getFile().getLocation().segment(0)))
```

Comment:
We shouldn't treat the first segment as a project name. It is error prone.

Reply:
Git project is always a root folder

Reply:
Can it help?
 https://github.com/eclipse/che/blob/6e9825c699d34eef6b8bf36fc0194ff871cbee84/plugins/plugin-git/che-plugin-git-ext-git/src/main/java/org/eclipse/che/ide/ext/git/client/GitUtil.java#L48

Reply:
done